knowledge-base/actions/artiomtr/jest-coverage-report-action/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: "jest coverage report "
+github-token:
+  action-input:
+    input: github-token
+    is-default: true
+  permissions:
```

Comment:
@RksRocks, I think there is:
- `pull-request: read` permission to check code coverage of pr
- `issues: write` to comment the code-coverage to pr

@varunsh-coder please review the permissions for this action, as some additional permission is needed for `on: push` trigger. check [here](https://github.com/ArtiomTr/jest-coverage-report-action#pull-request-number), [here](https://github.com/ArtiomTr/jest-coverage-report-action/search?q=octokit.rest)

Reply:
> @varunsh-coder please review the permissions for this action, as some additional permission is needed for `on: push` trigger. check [here](https://github.com/ArtiomTr/jest-coverage-report-action#pull-request-number), [here](https://github.com/ArtiomTr/jest-coverage-report-action/search?q=octokit.rest).

@varunsh-coder please review it's permission, I think it is related to this issue #1161